packages/api/src/external/commonwell/document/document-query.ts

```diff
@@ -135,6 +136,8 @@ export async function queryAndProcessDocuments({
   if (!isCwEnabledForCx) return interrupt(`CW disabled for cx ${cxId}`);
   const isCwQueryEnabled = await isFacilityEnabledToQueryCW(facilityId, patientParam);
   if (!isCwQueryEnabled) return interrupt(`CW disabled for facility ${facilityId}`);
+  const isCwEnabledForPatient = isCommonwellEnabledForPatient(patientParam);
+  if (!isCwEnabledForPatient) return interrupt(`CW disabled for patient, skipping...`);
```

Comment:
Maybe we just replace this by a call to `validateCWEnabled()`?

Reply:
Done.